### ISSUE TITLE
I am a bit unsure about the absolute value for the total income

### ISSUE THREAD

**Issue by Koealaquillage** [Ali-R-dev/expense-tracker#1]
Hey Ali,

Maybe I misread, but it seems that in that line you take the absolute value of the positive transactions for the total income. I am not sure is either the absolute value is necessary since everything is positive, or if you wanted to actually accept negative transactions as well but keep total income as  positive value. Can you tell me?

https://github.com/Ali-R-dev/expense-tracker/blob/acf0fc8b6c128d7228f8997428b84c6bd195bc88/src/App.jsx#L19

**Comment by Ali-R-dev** [comment]
Yes you are right .As i am checking for positive values in array, i do not need math.abs() . As only positive values are incomes.
